src/applications/personalization/dashboard/tests/containers/DashboardAppNew.unit.spec.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
```

Comment:
while this is an entirely new test file, it's basically a cut-and-paste job of the existing DashboardApp spec file